react_thervu_application/src/control/input/input.component.js

```diff
@@ -3,27 +3,27 @@ import PropTypes from 'prop-types';
 import classes from '../input/input.css';
```

Comment:
Since the input.css file is in the same directory of input.component.js, import as './input.css'

Reply:
Okay